tools/jenkins/compiler-common.sh

```diff
@@ -48,44 +46,37 @@ if [ "$PKGOS" = "OSX" ]; then
 
                 # older version, using clang-3.4
                 CC=clang-mp-3.4
-                CXX="clang++-mp-3.4 -std=c++14"
-                CXX17="clang++-mp-3.4 -std=c++1z"
+                CXX="clang++-mp-3.4 -std=c++1z"
                 GXX=g++-mp-4.9
                 OBJECTIVE_CC=$CC
                 OBJECTIVE_CXX=$CXX
                 ;;
             *)
                 # newer OS X / macOS version link with libc++ and can use the system clang
                 CC=clang
-                CXX="clang++ -std=c++14"
-                CXX17="clang++ -std=c++1z"
+                CXX="clang++ -std=c++1z"
                 OBJECTIVE_CC=$CC
                 OBJECTIVE_CXX=$CXX
                 ;;
         esac
     elif [ "$COMPILER" = "clang-omp" ]; then
         # newer version (testing) using clang-4.0
         CC=clang-mp-4.0
-        CXX="clang++-mp-4.0 -stdlib=libc++ -std=c++14"
-        CXX17="clang++-mp-4.0 -stdlib=libc++ -std=c++1z"
+        CXX="clang++-mp-4.0 -stdlib=libc++ -std=c++1z"
```

Comment:
ok. done. Given this statement, I've also changed the 3.4 reference above on line 49 to 5.0. I don't know if that path is ever used anymore, but changing that seems consistent with your comment here.

It might be nice to clean up the number of compilers and paths we have here to match the minimal Mac environment we want to support going forward. I don't really know anything about Mac so I'm not really a good person to clean this up.

Reply:
> match the minimal Mac environment we want to support going forward

RB-2.6 should work with macOS 10.13+ (lowest supported by the Qt version we use).